packages/alfa-rules/test/sia-r30/rule.spec.tsx

```diff
@@ -0,0 +1,147 @@
+import { h } from "@siteimprove/alfa-dom";
+import { None, Option } from "@siteimprove/alfa-option";
+import { test } from "@siteimprove/alfa-test";
+
+import R30, { Outcomes } from "../../src/sia-r30/rule";
+
+import { evaluate } from "../common/evaluate";
+import { oracle } from "../common/oracle";
+import { cantTell, failed, inapplicable, passed } from "../common/outcome";
+
+test(`evaluate() passes when some input rule passes`, async (t) => {
+  const target = <audio src="foo.mp3" />;
+  const transcript = <div>Hello</div>;
+
+  const document = h.document([target, transcript]);
+
+  t.deepEqual(
+    await evaluate(
+      R30,
+      { document },
+      oracle({
+        "is-audio-streaming": false,
+        "is-playing": true,
+        transcript: Option.of(transcript),
+      })
+    ),
+    [passed(R30, target, { 1: Outcomes.HasTextAlternative })]
+  );
+});
+
+test(`evaluate() passes when some input rule passes`, async (t) => {
```

Comment:
It could be nice to have different names for this and the previous test (line 11) as it's easier to distinguish them if only one fails in case of a change